features/admin.core.v1/components/add-certificate-form.tsx

```diff
@@ -42,17 +53,14 @@ interface AddApplicationCertificateFormProps extends TestableComponentInterface
 /**
  * Add Application certificate form component.
  *
- * @return {ReactElement}
+ * @returns ReactElement
  */
-export const AddApplicationCertificateFormComponent: React.FunctionComponent<AddApplicationCertificateFormProps> = (
-    props: AddApplicationCertificateFormProps): ReactElement => {
-
-    const {
-        triggerCertificateUpload,
-        onSubmit,
-        setShowFinishButton,
-        ["data-testid"]: testId
-    } = props;
+export const AddCertificateFormComponent: React.FunctionComponent<AddCertificateFormProps> = ({
+    triggerCertificateUpload,
+    onSubmit,
+    setShowFinishButton,
+    ["data-componentid"]: _componentId = "add-application-certificate-form"
```

Comment:
```suggestion
    ["data-componentid"]: _componentId = "add-application-certificate-form"
```
Need to remove the application part, as this is a common component

Reply:
Refer tests:
https://github.com/wso2-enterprise/asgardeo-tests/blob/55515ae3ea6fe1a14b64ebf1ac2ca25c210af7b9/cypress/page-objects/console/develop/applications/application-create.ts#L35